package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "appengine-cloudtasks",
+  "description": "Google App Engine Cloud Tasks example.",
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "private": true,
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "files": [
+    "*.js"
+  ],
+  "dependencies": {
+    "@google-cloud/tasks": "^3.0.0",
+    "body-parser": "^1.18.3",
+    "express": "^4.16.3"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^8.0.0",
+    "uuid": "^8.0.0"
```

Comment:
is uuid only a dev dependency? I think it should be a regular dependency. Similarly, do you need mocha and chai? I we don't have any tests for this code.

Reply:
Yes, I have already updated this file. We don't need mocha an chai.

Reply:
I am not really comfortable with having this in the root folder. Can we move this file into the .github/workflow directory? 
The reason is security and maintenance: this is not s JS project and we don't depend on JS libraries for the actual lib we publish. This is should be clearly only used for the github workflow

Reply:
Thanks for that Andi! I moved it into **workflows** folder.